api/src/user/schemas/user.schema.ts

```diff
@@ -50,6 +50,16 @@ export class UserStub extends BaseSchema {
   })
   password: string;
 
+  @Prop([{ type: MongooseSchema.Types.ObjectId, ref: 'Role' }])
+  roles: unknown;
+
+  @Prop({
+    type: MongooseSchema.Types.ObjectId,
+    ref: 'Attachment',
+    default: null,
```

Comment:
To check if the default reference to nul which is an object.

Reply:
Don't face an issue with the scenario tested from my side but, i think that it's better to investigate more about it. I have created a new issue with a short description for the mentioned bug https://github.com/Hexastack/Hexabot/issues/558